platform/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -57,8 +62,90 @@ const Tooltip = ({
 
   const isOpen = (isSticky || isActive) && !isDisabled;
 
+  useEffect(() => {
+    if (isOpen && parentRef.current && tooltipRef.current) {
```

Comment:
Nice! 

Can you please add a comment on what is happening for future devs, newX, newY, parent ...

Reply:
if (!isOpen || !parentRef.current || !tooltipRef.current){
  return
}

--> less indentation --> more clear